processing/src/main/java/io/druid/segment/filter/LikeFilter.java

```diff
@@ -52,10 +53,10 @@ public LikeFilter(
   @Override
   public ImmutableBitmap getBitmapIndex(BitmapIndexSelector selector)
   {
-    if (extractionFn == null && likeMatcher.getSuffixMatch() == LikeDimFilter.LikeMatcher.SuffixMatch.MATCH_EMPTY) {
+    if (directPrefixMatchable()) {
```

Comment:
"directPrefixMatchable" is unclear to me. Maybe "dimensionEqualsPrefix"? Then the comments on lines 57 and 88 could be removed